### ISSUE TITLE
.phedhistory file names changed

### ISSUE THREAD

**Issue by liberatorcnx** [pheditor/pheditor#68]
Every time I save a file, the file names in .phedhistory folder ALL change to the filename that I just saved.  For example, I save file1.php - now all files in .phedhistory folder are called file1.php.(0 thru 4).  This happens even if I sequentially save file1.php, file2.php etc...  They all end up with the name of the last file saved.  The files that are saved are correct - just the name is wrong.

I assume that the intention is to save the last 5 files saved BEFORE changes which is really useful, but not preserving the file name is not useful...

**Comment by hamidsamak** [comment]
Hi @liberatorcnx 
The history directory is used to keep last revisions of edited files for history and diff purposes. It is not completed and implemented yet and just keep files in that directory. The file names from 0 to 4 is due to configuration when ```MAX_HISTORY_FILES``` is set to 5. It means keeping five last revisions of file1.php with naming zero to 4.

**Comment by hamidsamak** [comment]
You're right, saving a history of renamed files is not implemented in the script. I'll keep this feature in the development list.
Thanks for report
